tls/extensions/s2n_client_psk.c

```diff
@@ -166,8 +166,9 @@ static int s2n_client_psk_send(struct s2n_connection *conn, struct s2n_stuffer *
         /* Write obfuscated ticket age */
```

Comment:
Per your callout, I think the change probably should be made to s2n_x509_validator.c.

Reply:
I did make the change. Not sure If I used to the right macro for return.